doc/rose-rug-suite-writing-tutorial.html

```diff
@@ -509,21 +509,13 @@ <h3 class="alwayshidden">suite.rc Contents</h3>
 #!jinja2
 [cylc]
     UTC mode = True # Ignore DST
-    [[event hooks]]
-        timeout handler = rose suite-hook --mail --shutdown
-        timeout = P3D
+    abort if any task fails = True
 [scheduling]
 [runtime]
     [[root]]
-        command scripting = rose task-run --verbose
-        [[[event hooks]]]
-           succeeded handler = rose suite-hook
-           failed handler = rose suite-hook --shutdown
-           submission failed handler = rose suite-hook --shutdown
-           submission timeout handler = rose suite-hook --mail
-           execution timeout handler = rose suite-hook --mail
-           submission timeout = P1D
-           execution timeout = PT3H
+        script = rose task-run --verbose
+        [[[events]]]
+            mail events = submission timeout, execution timeout
```

Comment:
is there a shutdown events entry now? If so, you need to put "submit-fail" and "fail" entries under that